packages/odyssey-react/src/components/Table/Table.module.scss

```diff
@@ -119,27 +104,26 @@
 
 /* Sort directions */
 
+.indicator {
```

Comment:
Same thoughts here as above with Link. Should we be more specific?

Reply:
I think for link it's a simple component that could take more than one type of indicator. 
For table its complex enough it would benefit from less ambiguity.